doc/system_specific_setup.rst

```diff
@@ -135,10 +138,11 @@ Install pytest for running python tests
 
    > sudo apt-get install python-pytest
 
-You can now run the PSyclone python tests
-::
+You can now run the PSyclone python tests:
+
+.. parsed-literal::
 
-   > cd PSyclone_trunk/src/tests
+   > cd PSyclone-\ |release|\ /src/psyclone/tests
    > py.test
 
 Documentation
```

Comment:
It is not possible to build the documentation without first installing using setup.py as this creates the version numbers. The documentation needs to be updated (as does the doc README file).